packages/application-config/src/sanitize-svg.ts

```diff
@@ -8,9 +8,10 @@ const DOMPurify = createDOMPurify(jsdom.window);
 export default function sanitizeSvg(data: string) {
   return DOMPurify.sanitize(data, {
     USE_PROFILES: { svg: true },
+    RETURN_DOM: true,
     FORBID_ATTR: [
       // To avoid injection by using `style="filter:url(\"data:image/svg+xml,<svg`
       'style',
     ],
-  });
+  }).innerHTML;
```

Comment:
Could you perhaps explain what this changes are meant for?

Reply:
Yes sorry. We already changed this in the backend, due to some security findings.

<img width="795" alt="image" src="https://user-images.githubusercontent.com/1110551/198013813-050d1103-4995-42d9-9d77-3da96a7d603d.png">

https://research.securitum.com/mutation-xss-via-mathml-mutation-dompurify-2-0-17-bypass/

Reply:
Thanks for the explanation 👍

Reply:
Wowzers, thanks for explaining 🙏 I guess we should remember to change that then in the `<InlineSvg>` component from ui-kit too (probably as part of the upcoming svg changes)

Reply:
Yes, good point!